app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -237,6 +240,44 @@
       },
       'keyword': function(predicate, item){
         return !!~item.name.toLowerCase().indexOf(predicate);
+      },
+      'light': function(predicate, item){
+        if (predicate.length === 0 || item.primStat == undefined) return false;
+        if (item.primStat.statHash != '3897883278' && item.primStat.statHash != '368428387') return false;
```

Comment:
Black spindle and malice most likely.
On Fri, Oct 9, 2015 at 8:27 AM kyle shay <notifications@github.com> wrote:

> In app/scripts/shell/dimSearchFilter.directive.js
> <https://github.com/DestinyItemManager/DIM/pull/284#discussion_r41642987>:
>
> > @@ -237,6 +240,44 @@
> >        },
> >        'keyword': function(predicate, item){
> >          return !!~item.name.toLowerCase().indexOf(predicate);
> > +      },
> > +      'light': function(predicate, item){
> > +        if (predicate.length === 0 || item.primStat == undefined) return false;
> > +        if (item.primStat.statHash != '3897883278' && item.primStat.statHash != '368428387') return false;
>
> what are these values ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DestinyItemManager/DIM/pull/284/files#r41642987>.
>